.github/workflows/render-rmarkdown.yaml

```diff
@@ -0,0 +1,48 @@
+on:
+  push:
+    paths:
+      - 'README.Rmd'
+
+name: Render README
+
+jobs:
+  build:
+    runs-on: macOS-latest
```

Comment:
Why using MACos here? Maybe try with ubuntu?

Reply:
🤷‍♂️ part of `r-lib` template